subsys/net/lib/shell/iface.c

```diff
@@ -817,6 +818,82 @@ static int cmd_net_default_iface(const struct shell *sh, size_t argc, char *argv
 	return 0;
 }
 
+#if defined(CONFIG_ETH_PHY_DRIVER)
+static int cmd_net_link_speed(const struct shell *sh, size_t argc, char *argv[])
+{
+	int idx = get_iface_idx(sh, argv[1]);
+	const struct device *phy_dev;
+	bool half_duplex = false;
+	uint16_t user_input_spd;
+	struct net_if *iface;
+	uint16_t speed = 0;
+
+	if (argc < 3) {
+		PR_WARNING("Usage: net iface set_link <index>"
```

Comment:
I think there should be a space between index and speed examples. So
```suggestion
		PR_WARNING("Usage: net iface set_link <index> "
```